.github/workflows/ci-cd-test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
   deploy-slack-notifier-test:
     needs: [deploy-test, check-for-changes]
-    uses: ./.github/workflows/DeployFunction.yml
+    uses: ./.github/workflows/action-deploy-function.yml
     if: ${{ needs.check-for-changes.outputs.hasSlackNotifierChanges == 'true' }}
```

Comment:
Ser at du ikke har rørt denne delen av koden i denne pren, men har noen spørsmål:
1. Er ikke "needs check-for-changes" implisitt gjennom if sjekken
2. Trenger du "needs." først i if'en?

Reply:
1. Vi er nødt til å ha den i needs for å kunne referere til den
2. ja stemmer. Vi avhenger av en annen job og kan da bruke "needs" for å referere til den